iter_test.go

```diff
@@ -0,0 +1,119 @@
+//go:build go1.23
+// +build go1.23
+
+package rill
+
+import (
+	"errors"
+	"fmt"
+	"iter"
+	"testing"
+	"time"
+
+	"github.com/destel/rill/internal/th"
+)
+
+func rangeInt(from, to int) iter.Seq[int] {
+	return func(yield func(i int) bool) {
+		for i := from; i < to; i++ {
+			if !yield(i) {
+				break
+			}
+		}
+	}
+}
+
+func TestToSeq2(t *testing.T) {
+	t.Run("errors", func(t *testing.T) {
+		in := FromSeq(rangeInt(0, 20), nil)
+		expectedErrs := []error{fmt.Errorf("err15"), fmt.Errorf("err18")}
```

Comment:
I'm sorry but using fmt.Errorf for creating error without sprintf formatting is awkward

Here and everywhere, it should be "errors.New"